requirements/dev-bookworm-requirements.txt

```diff
@@ -185,6 +185,10 @@ isort==5.10.1 \
     --hash=sha256:6f62d78e2f89b4500b080fe3a81690850cd254227f27f75c3a0c491a1f351ba7 \
     --hash=sha256:e8443a5e7a020e9d7f97f1d7d9cd17c88bcb3bc7e218bf9cf5095fe550be2951
     # via -r requirements/dev-sdw-requirements.in
+jinja2==3.0.2 \
+    --hash=sha256:827a0e32839ab1600d4eb1c4c33ec5a8edfbc5cb42dafa13b81f182f97784b45 \
+    --hash=sha256:8569982d3f0889eed11dd620c706d39b60c36d6d25843961f33f77fb6bc6b20c
+    # via -r requirements/requirements.in
```

Comment:
since this is a production dependency, we'll have to add it to the `securedrop-builder` repo as well ([docs](https://github.com/freedomofpress/securedrop-builder#updating-python-wheels)). In the meantime, we'll see the corresponding CI failures.

Reply:
The hash: `827a0e32839ab1600d4eb1c4c33ec5a8edfbc5cb42dafa13b81f182f97784b45` matches the one added in https://github.com/freedomofpress/securedrop-builder/pull/407 :white_check_mark: